src/microbe_stage/Microbe.cs

```diff
@@ -179,12 +179,26 @@ public override void _Ready()
 
         Membrane = GetNode<Membrane>("Membrane");
         OrganelleParent = GetNode<Spatial>("OrganelleParent");
-        engulfAudio = GetNode<AudioStreamPlayer3D>("EngulfAudio");
         bindingAudio = GetNode<AudioStreamPlayer3D>("BindingAudio");
         movementAudio = GetNode<AudioStreamPlayer3D>("MovementAudio");
 
         cellBurstEffectScene = GD.Load<PackedScene>("res://src/microbe_stage/particles/CellBurstEffect.tscn");
 
+        engulfAudio = new HybridAudioPlayer
+        {
+            Stream = GD.Load<AudioStream>("res://assets/sounds/soundeffects/engulfment.ogg"),
+            Positional = !IsPlayerMicrobe,
+            Bus = "SFX",
+        };
+
+        AddChild(engulfAudio);
```

Comment:
engulfAudio could use _Ready to determine which player type to create as that should trigger here after the properties have been set.